mkdocs.yml

```diff
@@ -235,6 +235,7 @@ nav:
       - OpenCTI: cti/features/integrations/opencti.md
       - Splunk: cti/features/integrations/splunk.md
       - Anomali ThreatStream: cti/features/integrations/anomali.md
+      - PaloAlto Cortex XSOAR: cti/features/integrations/palolto_xsoar.md
```

Comment:
You should also add this link in the “XDR” section

Reply:
Are you sure for the "XDR" section? After a little analysis, It seems "TIP" was the more suitable.